src/commands/createWebApp/createWebApp.ts

```diff
@@ -5,16 +5,14 @@
 
 import { AzureParentTreeItem, IActionContext, ICreateChildImplContext } from "vscode-azureextensionui";
 import { SubscriptionTreeItem } from '../../explorer/SubscriptionTreeItem';
-import { WebAppTreeItem } from "../../explorer/WebAppTreeItem";
 import { ext } from "../../extensionVariables";
 
 export async function createWebApp(context: IActionContext & Partial<ICreateChildImplContext>, node?: AzureParentTreeItem | undefined): Promise<void> {
     if (!node) {
         node = <AzureParentTreeItem>await ext.tree.showTreeItemPicker(SubscriptionTreeItem.contextValue, context);
     }
 
-    const newSite: WebAppTreeItem = <WebAppTreeItem>await node.createChild(context);
-    newSite.promptToDeploy(context);
```

Comment:
I don't think you meant to remove this line?

Reply:
Yeah, you're right =_= I'm being stupid, I was thinking that we didn't use `newSite` for anything for some reason